susemanager/src/mgr_bootstrap_data.py

```diff
@@ -731,6 +731,10 @@
     "salt-minion"
 ]
 
+PKGLISTDEBIAN11 = [
+    "venv-salt-minion"
```

Comment:
Tested on a container:
```
root@fc0b9b4456ef:/# dpkg -i venv-salt-minion_3002.2-35.1.uyuni_amd64.deb 
Selecting previously unselected package venv-salt-minion.
(Reading database ... 7094 files and directories currently installed.)
Preparing to unpack venv-salt-minion_3002.2-35.1.uyuni_amd64.deb ...
Unpacking venv-salt-minion (3002.2-35.1.uyuni) ...
dpkg: dependency problems prevent configuration of venv-salt-minion:
 venv-salt-minion depends on logrotate; however:
  Package logrotate is not installed.
 venv-salt-minion depends on systemd; however:
  Package systemd is not installed.

dpkg: error processing package venv-salt-minion (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 venv-salt-minion
```
Having those two packages absent on containers is OK, but all other Debian installations, even the most minimal setups will have `systemd` and `logrotate`